### PR TITLE
Feature/package wizard UI update

### PR DESCRIPTION
**Pull request by cfu-work**
## Jira Link

[SOLMP-23](https://jira.unity3d.com/browse/SOLMP-23)
[SOLMP-21](https://jira.unity3d.com/browse/SOLMP-21)
[SOLMP-25](https://jira.unity3d.com/browse/SOLMP-25)
[SOLMP-27](https://jira.unity3d.com/browse/SOLMP-27)

## Description

- Change toggles to left side toggles
- Remove does it score
- Add tooltips
- Reorder elements
- Separate elements into their own uxml

## Tech Notes

N/A

## Screenshots

image
image

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Refactor of Systems
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

## Checklist
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
- [N/A] Develop has been merged into this branch
- [x] Project builds and runs in Unity Editor
- [N/A] Updated package version

### CODE REVIEW

File: Packages/com.miniproject.packagewizard/Editor/Scripts/EditorWindow/PackageDetails.uxml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<ui:UXML xmlns:ui="UnityEngine.UIElements" xmlns:uie="UnityEditor.UIElements" editor-extension-mode="False">
+    <Style src="project://database/Packages/com.miniproject.packagewizard/Editor/Scripts/EditorWindow/PackageWizard.uss?fileID=7433441132597879392&amp;guid=0c061ff59c04373489742356252f97ea&amp;type=3#PackageWizard" />
+    <ui:VisualElement name="PackageDetails">
+        <ui:Label text="Package Details" name="PackageDetailsTitle" class="section-title" />
+        <ui:TextField picking-mode="Ignore" label="Package Name" value="filler text" name="package-name" style="border-left-width: 6px; border-right-width: 6px; border-top-width: 6px; border-bottom-width: 6px; margin-left: 0; margin-right: 0; margin-top: 0; margin-bottom: 0; padding-left: 0; padding-right: 0; padding-top: 0; padding-bottom: 0;" />
```
Comment (AlexBedardReidU3D) [inline Packages/com.miniproject.packagewizard/Editor/Scripts/EditorWindow/PackageDetails.uxml]: Not urgent, but we should be aiming to move any in-line styles into USS classes, as that should help reduce the clutter of the UXML as well as make things easier to edit

Comment (cfu-work) [reply Packages/com.miniproject.packagewizard/Editor/Scripts/EditorWindow/PackageDetails.uxml]: Might as well get in the habit of it now!

Comment (nhayer-work) [reply Packages/com.miniproject.packagewizard/Editor/Scripts/EditorWindow/PackageDetails.uxml]: Yeah let's clean this up, It's excellent habit to get into. Inline sucks

Comment (cfu-work) [reply Packages/com.miniproject.packagewizard/Editor/Scripts/EditorWindow/PackageDetails.uxml]: I've updated the uxmls! I left one inline style in the PackageCreation.uxml for a button that will be used later.

**Pull request merged**